client/app/scripts/charts/nodes-chart-elements.js

```diff
@@ -157,18 +157,17 @@ class NodesChartElements extends React.Component {
         focused={node.get('focused')}
         highlighted={node.get('highlighted')}
         shape={node.get('shape')}
-        stack={node.get('stack')}
+        stacked={node.get('stack')}
         key={node.get('id')}
         id={node.get('id')}
         label={node.get('label')}
         labelMinor={node.get('labelMinor')}
         pseudo={node.get('pseudo')}
         rank={node.get('rank')}
-        dx={node.get('x')}
-        dy={node.get('y')}
-        scale={node.get('scale')}
+        x={node.get('x')}
+        y={node.get('y')}
+        size={100 * node.get('scale')}
```

Comment:
just commenting to check this is intentional as `100 * x` is quite a jump from `x` 👌

Reply:
Oh yes, the scale is a relative scale actually (normalized at `1`) and `100` corresponds to node base size, so I should replace it with the actual existing `NODE_BASE_SIZE` constant. Good catch! :)